### ISSUE TITLE
Documenting the code

### ISSUE THREAD

**Issue by PiotrSokol** [thomasdeneux/spikes#7]
Hi,

I've finally had time to take your code for a spin and I'm trying to test it on GCaMP6s. Using the standard calibrate function returns 'no events found' message, even when testing on the GCaMP6f data used in the spikefinder challenge. 
I know it's a matter of setting the hyperparams, since you very successfully ran it on GCamP6 in the spikefinder challenge (congrats!). I tried digging out the presets from the challenge code but they're hard to follow. 

Could you write a short add-on to the tutorial how to chose and set the parameters for different indicator dyes?

Best,
P.

**Comment by thomasdeneux** [comment]
Hello Piotr,

I recommend not using the autocalibration unless your data is really high-quality or you became an expert. In particular i did not use it for the SpikeFinder contest, but only MLspike alone.

I have added much information in the documentation, check the new version on github, it includes in particular all the additional code that i used for the SpikeFinder submission.

Best regards,

Thomas
